data/mt/retired/John-Fuller-54fe073e-e061-43aa-9299-42fe869102d0.yml

```diff
@@ -7,7 +7,8 @@ email: John.Fuller@mtleg.gov
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2022-12-31'
```

Comment:
@bfossen-ce this person is in the pull twice. They were in the house from 2019 until 01/02/2023. On that date they are beginning in the senate